tests/utils/test_yaml.py

```diff
@@ -13,7 +13,6 @@ def test_yaml() -> None:
     prod:
       type: postgres
       host: 127.0.0.1
-      # IMPORTANT: Make sure to quote the entire Jinja string here
```

Comment:
why was this removed?

Reply:
because the C yaml parser doesn't preserve comments